src/test/java/com/woowacamp/soolsool/core/statistics/domain/StatisticsTest.java

```diff
@@ -0,0 +1,32 @@
+package com.woowacamp.soolsool.core.statistics.domain;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+import com.woowacamp.soolsool.core.statistics.domain.vo.BrewType;
+import com.woowacamp.soolsool.core.statistics.domain.vo.Click;
+import com.woowacamp.soolsool.core.statistics.domain.vo.Impression;
+import com.woowacamp.soolsool.core.statistics.domain.vo.Region;
+import com.woowacamp.soolsool.core.statistics.domain.vo.SalePrice;
+import com.woowacamp.soolsool.core.statistics.domain.vo.SaleQuantity;
+import java.math.BigInteger;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("통계 단위 테스트")
+class StatisticsTest {
```

Comment:
```suggestion
@DisplayName("단위 테스트: Statistic")
class StatisticsTest {
```

Statistic의 이름과 다른 테스트와 형식이 일치하면 좋을 것 같아요.